modules/apps/29-fee/ibc_middleware.go

```diff
@@ -404,7 +404,16 @@ func (im IBCMiddleware) OnChanUpgradeAck(ctx sdk.Context, portID, channelID, cou
 
 // OnChanUpgradeOpen implements the IBCModule interface
 func (im IBCMiddleware) OnChanUpgradeOpen(ctx sdk.Context, portID, channelID string, order channeltypes.Order, connectionHops []string, version string) {
-	// call underlying app's OnChanUpgradeOpen callback.
+	_, err := types.MetadataFromVersion(version)
```

Comment:
discarding the `versionMetadata`, unless anyone thinks the fee version on it should be checked against `types.Version` again, in which case we need to panic inside here... which is undesirable imo